packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -72,43 +72,14 @@ const getQueryableFields = (object: ObjectType): Field[] => (
   Object.values(object.fields).filter(isQueryableField)
 )
 
-const getFieldNamesForQuery = async (field: Field): Promise<string[]> => (
-  await isNameField(field)
-    ? Object.keys((await field.getType() as ObjectType).fields)
-    : [await apiName(field, true)]
-)
-
-/**
- * Build a set of queries that select records.
- *
- * @param typeName The name of the table to query from
- * @param fields The names of the fields to query
- * @param conditionSets Each entry specifies field values used to match a specific record
- * @param maxQueryLen returned queries will be split such that no single query exceeds this length
- */
-export const buildSelectQueries = async (
-  typeName: string,
-  fields: Field[],
-  conditionSets?: Record<string, string>[],
-  maxQueryLen = MAX_QUERY_LENGTH,
-): Promise<string[]> => {
-  const fieldsNameQuery = (
-    await awu(fields).flatMap(getFieldNamesForQuery).toArray()
-  ).join(',')
-  const selectStr = `SELECT ${fieldsNameQuery} FROM ${typeName}`
-  if (conditionSets === undefined || conditionSets.length === 0) {
-    return [selectStr]
-  }
-  return conditionQueries(selectStr, conditionSets, maxQueryLen)
-}
-
 const buildQueryStrings = async (
   typeName: string,
   fields: Field[],
   ids?: string[]
-): Promise<string[]> => (
-  buildSelectQueries(typeName, fields, ids?.map(id => ({ Id: `'${id}'` })))
-)
+): Promise<string[]> => {
+  const fieldNames = await awu(fields).flatMap(getFieldNamesForQuery).toArray()
```

Comment:
When using awu/iterable functions it's prefered to new line each method.
```suggestion
  const fieldNames = await awu(fields)
    .flatMap(getFieldNamesForQuery)
    .toArray()
```